packages/grid/x-data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('<DataGrid /> - Layout & Warnings', () => {
     columns: [{ field: 'brand' }],
   };
 
-  describe('immutable rows', () => {
+  describe('Immutable rows', () => {
```

Comment:
This is a sub-suite and in the possible error message, this part will end up in the middle of a sentence.
Does it make sense to capitalize it? 🤔